### DISCUSSION TITLE
%E4%B8%BB%E6%97%A5%E8%B3%87%E6%96%99/1131117%E4%B8%BB%E6%97%A5%E7%B0%A1%E5%A0%B1

### DISCUSSION THREAD

**Discussion by giscus** [cit-hall-43/cit-hall-43.github.io#10] [Q&A]
# %E4%B8%BB%E6%97%A5%E8%B3%87%E6%96%99/1131117%E4%B8%BB%E6%97%A5%E7%B0%A1%E5%A0%B1

讚美主

https://cit-hall-43.github.io/%E4%B8%BB%E6%97%A5%E8%B3%87%E6%96%99/1131117%E4%B8%BB%E6%97%A5%E7%B0%A1%E5%A0%B1.html

**Comment by cit-hall-43** [comment]
補充本338

**Comment by cit-hall-43** [comment]
鄭弟兄 分享交通禱讀卡
罪奴，主用重價買我們，主是第一個奴僕的榜樣，要開通耳朵，犧牲的靈，不是轄管，要作眾人的榜樣，保羅是耶穌基督的奴僕，李弟兄也是
知道是奴僕就不一樣，幫門徒洗腳，服事人，聽話順服

很多聖徒被拐跑（拐走）、被我們打和傷（故意和無意）
